Dockerfile

```diff
@@ -2,9 +2,16 @@ FROM ruby:2.6.5-alpine AS middleman
 
 RUN apk add --update --no-cache npm git build-base
 
-ADD . .
+COPY docs/Gemfile docs/Gemfile.lock /
 
-RUN cd docs && bundle install --jobs=4 && bundle exec middleman build --build-dir=../public && cd -
+RUN bundle install --jobs=4
+
+COPY docs /docs
+COPY public /public
+COPY swagger /swagger
+
+WORKDIR docs
+RUN bundle exec middleman build --build-dir=../public && cd -
```

Comment:
What's the poiunt of `cd -`?

Reply:
No point, it was just left over from before. Will trim

Reply:
can we stick `bundle exec rake rswag:specs:swaggerize` somwhere where here?

Reply:
The generated file `swagger/public_v1/api_spec.json` is checked into the repo, so this would mean that we could potentially have an openapi spec that's not up-to-date in the repo, but generate a different set of docs.

Sounds to me like what we outta do is run `bundle exec rake rswag:specs:swaggerize` and fail the build if it turns out `swagger/public_v1/api_spec.json` isn't up-to-date. Sounds like that belongs in another PR.